release-notes.md

```diff
@@ -0,0 +1,3 @@
+## AbilityToGroom
+### zib-AbilityToGroom
+* `.value[x]:valueCodeableConcept` ; textual ; Refined guidance for Grooming ; ZIB-1114
```

Comment:
What kind of guidance is given in the profile?  I only see a definition.

I would say that this profile does not contain any significant changes.

Reply:
Changed to No significant changes.